src/api/api.ts

```diff
@@ -1 +1,7 @@
-export const BASE_URL = 'https://mate.academy/students-api';
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getAllUsers() {
+  const response = await fetch(`${BASE_URL}/users`);
+
```

Comment:
Throw an error in case something goes wrong.
```suggestion

  if (!response.ok) {
    throw new Error(`${response.status} - ${response.statusText}`);
  }

```